docs/architecture/introduction.rst

```diff
@@ -100,11 +100,10 @@ of CLI commands.
 Model Registry
 ==============
 
-The Model Registry is a way to group together conceptually related checkpoints (including ones
-across different experiments), storing metadata and longform notes about a model, and retrieving the
-latest version of a model for use or futher development. The Model Registry can be accessed through
-the WebUI, Python SDK, REST API, or CLI, though the WebUI has some features that the others are
-missing.
+The Model Registry is a way to group conceptually-related checkpoints (including ones across
```

Comment:
```suggestion
The Model Registry is a way to group conceptually related checkpoints (including ones across
```
https://www.merriam-webster.com/words-at-play/6-common-hypercorrections-and-how-to-avoid-them/hyphenating-ly-adverbs

Reply:
resolved